patterns/base/table/table.twig

```diff
@@ -0,0 +1,18 @@
+<table class="{{table.class}}">
+  <thead class="{{table.head.class}}">
+    <tr class="base--tr">
```

Comment:
maybe a table won't have a tbody or thead...maybe they won't be in that order....or well... I just don't know :-)

I'm trying to plan this for the scenarios we can't think of yet...